src/arch/amdgpu/vega/pagetable.hh

```diff
@@ -63,7 +63,7 @@ BitUnion64(PageTableEntry)
     Bitfield<2> c;
     Bitfield<1> s;
     Bitfield<0> v;
-EndBitUnion(PageTableEntry)
+ButUnionEnd(PageTableEntry)
```

Comment:
The sed cmd had a typo

Reply:
Andreas Sandberg has found a mistake in the clang format. The sed is not needed anymore. 
Thanks anyway!